docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -32,7 +32,8 @@ The router service is the initial entrypoint for all requests. It is HTTP centri
 
 | Selector           | Defaultable | Values                                    | Description                          |
 |--------------------|-------------|-------------------------------------------|--------------------------------------|
-| `trace_id`         | Yes         | `open_telemetry`\|`datadog`              | The trace ID                         |
+| `trace_id`         | Yes         | `open_telemetry`\|`datadog`               | The trace ID                         |
+| `operation_name`   | Yes         | `string`|`hash`                           | The operation name from the query    |
 | `studio_operation_id` | Yes      | `true`|`false`                            | The Apollo Studio operation id       |
```

Comment:
Unrelated but does this make sense? Operation ID as a boolean?

Reply:
The values listed here are not the value that the selector will have. It's just the configuration value, you can either enable it (`true`)  or disable it (`false`).